config/v1.5/aws-k8s-cni.yaml

```diff
@@ -76,6 +76,10 @@ spec:
                     operator: In
                     values:
                       - amd64
+                  - key: eks.amazonaws.com/compute-type
+                    operator: NotIn
+                    values:
+                      - fargate
```

Comment:
@nithu0115 is this addressing a specific problem you've run into with customers? I closed a similar issue (#755) stating that we didn't want to place EKS-specific configuration values into the CNI plugin's YAML manifests because CNI can and should run everywhere, not just EKS. Do you disagree with that?

Reply:
@jaypipes I agree up to a point, but the source of truth for customer to upgrade to latest CNI version (to 1.5.5) from default 1.5.3 version is still GitHub as per EKS public documentation.  

IMO, I still think we should have this node affinity params to provide clean upgrade experience, and should have a strategy to remove (like introducing customer to helms charts) going forward.

Reply:
Chatted with @mogren about this some more this morning. He has convinced me that this won't do any harm, and although my objective is to keep EKS-specific items out of the VPC CNI configuration, this won't harm anyone running the VPC CNI plugin on non-EKS k8s on AWS